### DISCUSSION TITLE
What are reserved slots and how do I use them?

### DISCUSSION THREAD

**Discussion by Mooshay105** [Pryaxis/TShock#2996] [Q&A] [answered]
Same as the title.

**Comment by sgkoishi** [comment, marked as answer]
https://github.com/Pryaxis/TShock/blob/5d585bb891ef6ed855ec3692c36712e64be3d867/TShockAPI/Permissions.cs#L452-L453

There are some reserved/hidden positions for VIPs/admins to join the server when it is full. Giving players this permission will allow them to utilize this feature.

**Reply by Mooshay105** [reply]
Ok, thank you!
